src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship.UnitTests/Requests/WhenBuildingGetVacanciesRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.FindAnApprenticeship.InnerApi.Requests;
+
+namespace SFA.DAS.FindAnApprenticeship.UnitTests.Requests;
+
+public class WhenBuildingGetVacanciesRequest
+{
+    [Test, AutoData]
+        public void Then_The_Request_Url_Is_Correctly_Built(double lat, double lon, List<string> routes, int distance)
+        {
+            var actual = new GetVacanciesRequest(lat, lon, routes, distance);
+
+            actual.GetUrl.Should().Be($"/api/vacancies?lat={lat}&lon={lon}&routes={routes}&distance={distance}");

```

Comment:
im not sure this is doing exactly as you'd expect, id expect the GetUrl should be something like

```suggestion
            actual.GetUrl.Should().Be($"/api/vacancies?lat={lat}&lon={lon}&routes={routes}&routes={routes}&distance={distance}");

```
the to string on the list will just return the ToString of the type and not the values joined together.